doc/dev/mempool_accounting.rst

```diff
@@ -0,0 +1,56 @@
+What is a mempool?
+------------------
+A memory pool is a method for accounting the consumption of memory of
+a set of containers.
+Each memory pool tracks the number of bytes and items it contains.
+Memory pools are statically declared and serves the purpose of identifying
+memory related checks in BlueStore.
+
+Some common mempools that we can track
+--------------------------------------
+
+- bloom_filter: Ceph uses bloom filters to account for tracking objects already
```

Comment:
Suggest
    ``bloom_filter``: Ceph uses ...
and similarly for other list items to highlight that these are proper names.

Reply:
Formatted! Please take another look.